Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/profile/ProfilePage.tsx

```diff
@@ -9,15 +9,20 @@ import ProfileSubmissions from '../../components/profile/profile-submissions/Pro
 import { useUsers } from '../../hooks/use-users';
 import { makePrivate } from '../shared/make-private';
 import { setLayout } from '../shared/set-layout';
+import SpinningLoader from "../../components/guidelines/spinning-loader/SpinningLoader";
+import {flexCenterObjectStyles} from "../../utils/object-utils";
 
 const ProfilePage = () => {
-    const { profile, getProfile } = useUsers();
+    const { profile, getProfile, isLoading } = useUsers();
 
     useEffect(() => {
         (async () => {
             await getProfile();
         })();
     }, [ getProfile ]);
+    
+    if(isLoading) return <div style={{...flexCenterObjectStyles}}><SpinningLoader/></div>
```

Comment:
use ` { } ` for the `if` block. Also isn't better for this `<div style={{...flexCenterObjectStyles}}` to be inside the `SpinningLoader ` and pass the styles directly to it?

Reply:
to have content centered would require its parent to have center styling properties for its children, vice versa would not work as expected

Reply:
Ok, just fix the if block